vignettes/dashboard.Rmd

```diff
@@ -89,27 +89,22 @@ cran_incoming %>%
 # CRAN review workflow
 
 Your package will be stored in a different folder depending on its current state
-in the review process. The exact meaning of each folder is detailed in an 
-[article from the 
-R Journal](https://journal.r-project.org/archive/2018-1/cran.pdf) as well as in [another article from the same journal](https://journal.r-project.org/archive/2019-1/cran.pdf):
-
-- **inspect**: this is your first submission or the automated tests found an 
-error that requires human review.
-- **newbies**: a specific queue for the manual inspection of first time CRAN submissions.
-- **pending**: the CRAN maintainers are waiting for an action on your side. You 
-should check your emails!
-- **waiting**: packages for which the CRAN team waits for an answer from the maintainer.
+in the review process. The exact meaning of each folder is detailed in articles from 
+the R Journal in [2018](https://journal.r-project.org/archive/2018-1/cran.pdf) and [2019](https://journal.r-project.org/archive/2019-1/cran.pdf), and updated by a [2019 mailing list post](https://stat.ethz.ch/pipermail/r-package-devel/2019q1/003631.html) (and [confirmed in 2022](https://stat.ethz.ch/pipermail/r-package-devel/2022q2/008084.html)):
+
+- **inspect**: your package is awaiting manual inspection by the CRAN team, probably because automated tests found a problem that is likely to be a false positive
+- **newbies**: a specific queue for the manual inspection of first-time CRAN submissions.
+- **pending**: a [CRAN] team member has to do a closer inspection and needs more time.
```

Comment:
Why is CRAN between brackets here?

Reply:
doesn't really need to be in brackets. The original statement is quoted from Uwe Ligges's e-mail to the list and didn't say "CRAN", but as I'm not pretending to directly quote (and I don't think Uwe will mind our nearly-plagiarizing this one sentence) I guess it's unnecessary/adds confusion.